pom.xml

```diff
@@ -187,39 +197,21 @@
       </exclusions>
     </dependency>
 
-
     <dependency>
       <groupId>com.github.salesforce-marketingcloud</groupId>
       <artifactId>fuelsdk</artifactId>
       <version>${salesforce.marketing.cloud.version}</version>
     </dependency>
-
-    <dependency>
```

Comment:
why are these getting removed? were they never needed in the first place?

Reply:
They were not needed.  CFX library is already part of the SFMC jar.  Ref: https://github.com/salesforce-marketingcloud/FuelSDK-Java   . Check out the installation section of Readme.  CFX library is only required if you try to build the JAR from the source.

Reply:
Did you make sure removing this didn't break the existing sink? I see that the old 1.4.0 and 1.5.0 versions of the fuel library have cxf 3.1.2 as a dependency, whereas this used to be 3.1.9

Reply:
I personally did not test the sink but had Colgate team test it. Let me confirm again with them and get back to you.

Reply:
@albertshau  - I have forwarded the confirmation from Colgate that the SINK plugin that's built as part of 1.5.0 is working fine. This does not have dependency on the CXF jar's.